types.go

```diff
@@ -39,6 +41,8 @@ type Reshare struct {
 	Owner [20]byte `ssz-size:"20"`
 	// Owner nonce
 	Nonce uint64
+	// Amount in Gwei (https://eips.ethereum.org/EIPS/eip-7251)
+	Amount uint64
```

Comment:
A question (maybe I still don't know how DKG works?):
When you do reshare, you just reshare the validation keys, right? You don't recreate the deposit file?
Then why do you need amount?

Reply:
Yes, its a good question. The way we have spec - we need to change `BuildResult` to defer for `init`, `resign/reshare`